File: Ruby/season2/p010-011/p010.rb
# -*- coding: utf-8 -*-
=begin
　2017.5.2
　シンボルの作成と表示
=end

def main
  # シンボルと文字列の宣言
  symb0 = :po
  symb1 = :"popo"
  str = "popopo"
  
  # 表示
  puts "【表示】"
  p symb0
  p symb1
  p str

  # 操作
  puts "【操作】"
  p symb0.to_s + symb1.to_s + str
end

main

File: Python/season2/p007-010/p010.py
# coding: UTF-8
"""
　2018.4.6
　三角形の状態判定
"""

def triangle(a,b,c):
    if a+b < c or b+c < a or c+a < b:
      return "非三角形"
    elif a==b==c:
      return "正三角形"
    elif (a==b) or (b==c) or (c==a):
      return "二等辺三角形"
    else:
      return "三角形"

print( triangle(1,1,1) )
print( triangle(2,1,2) )
print( triangle(3,4,5) )
print( triangle(1,1,3) )